plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -694,18 +697,15 @@ private class DokkaDescriptorVisitor(
     }
 
     private fun AnnotationDescriptor.toAnnotation(): Annotations.Annotation? =
-        if(annotationClass?.visibility == Visibilities.PUBLIC)
-            DRI.from(annotationClass as DeclarationDescriptor)
-                .takeIf { it.classNames != "<ERROR CLASS>" }?.let {
-                    Annotations.Annotation(
-                        it,
-                        allValueArguments.map { it.key.asString() to it.value.toValue() }.filter {
-                            it.second != null
-                        }.toMap() as Map<String, AnnotationParameterValue>
-                    )
-                }
-        else
-            null
+        DRI.from(annotationClass as DeclarationDescriptor).let {
```

Comment:
There is absolutely no need for a scoping function here!

```kotlin
 private fun AnnotationDescriptor.toAnnotation2(): Annotations.Annotation? = Annotations.Annotation(
        DRI.from(annotationClass as DeclarationDescriptor),
        allValueArguments.map { it.key.asString() to it.value.toValue() }.filter {
            it.second != null
        }.toMap() as Map<String, AnnotationParameterValue>,
        annotationClass!!.annotations.hasAnnotation(FqName("kotlin.annotation.MustBeDocumented"))
    )
```

Reply:
Yeah, it probably is some leftover